cypress/support/e2e.ts

```diff
@@ -9,6 +9,7 @@ import '@cypress/code-coverage/support'
 import 'cypress-hardhat/lib/browser'
 
 import { Eip1193Bridge } from '@ethersproject/experimental/lib/eip1193-bridge'
+import TokenListJSON from '@uniswap/default-token-list'
```

Comment:
should we try to prevent this from being imported into product code? not sure how large it is